### DISCUSSION TITLE
Distinguish "broken" binding in the component tree

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#5308] [Ideas]
I've been copying and pasting custom-made components both within pages and across pages. However, especially when doing that across pages, data bindings in handlebars and javascripts will be broken into something like this:

> $("[c3ad1ed8ffc5443f1b841660e7233b980].[rows]")

Obviously all broken bindings must be fixed before the component works as expected. It will be very helpful to make element names with broken bindings distinguishable in the component tree (show them in red for example).

**Comment by melohagan** [comment]
Hi @dannyshin76 

I think this is a great idea! I've linked this discussion to an epic that is looking at better debugging within budibase in general:
https://github.com/Budibase/budibase/issues/4908
